packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -121,10 +121,10 @@ abstract class Route<T> {
   /// See [RouteSettings] for details.
   final RouteSettings settings;
 
-  /// The entries this route has placed in the overlay.
+  /// The overlay entries of this route.
   ///
-  /// These are typically populated by [install], added to the [Overlay] by the
-  /// [Navigator], and then removed by [dispose].
+  /// These are typically populated by [install], added to or removed from the
+  /// [Overlay] by the [Navigator]
```

Comment:
Let's word this a little stronger:

```suggestion
These are typically populated by [install]. The [Navigator] is in charge of adding them to and removing them from the [Overlay].
```